app/controllers/api/v3/exotel_call_sessions_controller.rb

```diff
@@ -70,13 +68,9 @@ def respond_in_plain_text(status, text = "")
     render plain: text, status: status
   end
 
-  def report_http_status
```

Comment:
I take it DataDog gives us these error code rates?

Reply:
Yup.